src/main/java/hudson/scm/browsers/AbstractSventon.java

```diff
@@ -59,7 +59,7 @@ public AbstractSventon(URL url, String repositoryInstance) throws MalformedURLEx
         // normalize
         repositoryInstance = repositoryInstance.trim();
 
-        this.repositoryInstance = repositoryInstance == null ? "" : repositoryInstance;
```

Comment:
Yes, it was never going to work - the code was introduced in https://github.com/jenkinsci/subversion-plugin/blame/master/src/main/java/hudson/scm/browsers/AbstractSventon.java